CHANGELOG.md

```diff
@@ -2,9 +2,22 @@
 
 ## Unreleased
 ### Improvements
-### Fixed
 
-## [4.2.2] - TBD
+## [4.4.0] - TBD
+### Improvements
```

Comment:
looks like something went wrong with rebase?

Reply:
Looks fine to me? But feel free to modify however you want and merge this.